modules/800-deckhouse-tools/images/web/werf.inc.yaml

```diff
@@ -0,0 +1,58 @@
+{{- $version := "v0.3.0" }}
```

Comment:
I suggest moving the version to version_map.yaml and using this version here and in the registry packages module https://github.com/deckhouse/deckhouse/blob/ffcf3fa9939986ba3f782c8f6a2c449f487ecde6/modules/007-registrypackages/images/d8/werf.inc.yaml#L1 -L0

To have one version in two different places, to update the version in these places at once

Reply:
and update to latest version please

Reply:
Moved.